Source/Core/RequestScheduler.js

```diff
@@ -383,6 +383,20 @@ define([
     }
 
     /**
+     * Clears the request scheduler before each spec.
+     */
+    RequestScheduler.clearForSpecs = function() {
```

Comment:
Explicitly mark this `@private`.

Also, would you rather have this here than in the tests so just it is easier maintained in case we add new properties?